### PR TITLE
Fixes the crime against language that was the juicer

### PR DESCRIPTION
**Pull request by Supermichael777**
Ow my language centers.

### CODE REVIEW

**Comment by ExcessiveUseOfCobblestone** [comment]
your desc needs a " at the end

**Comment by Supermichael777** [comment]
its not used anywhere buuuuuut

File: code/modules/food_and_drinks/kitchen_machinery/juicer.dm [hunk 1]
```diff
@@ -1,7 +1,7 @@
```
Comment (Iamgoofball) [inline code/modules/food_and_drinks/kitchen_machinery/juicer.dm]: get this shit out of my codebase's commit logs

**Review by Iamgoofball — CHANGES_REQUESTED** [review]

File: code/modules/food_and_drinks/kitchen_machinery/juicer.dm (hunk 1 above)
Comment (Supermichael777) [inline code/modules/food_and_drinks/kitchen_machinery/juicer.dm]: what do you mean "my coderbase"

and what do you actually want?

Comment (Supermichael777) [inline code/modules/food_and_drinks/kitchen_machinery/juicer.dm]: wait do you mean the empty first line?

**Pull request merged**